afs/media/js/views/components/reports/project.js

```diff
@@ -41,6 +41,40 @@ define([
             self.documentationCards = {};
             self.summary = params.summary;
 
+            self.childProjectTableConfig = {
+                ...self.defaultTableConfig,
+                columns: Array(4).fill(null)
+            };
+
+            self.collectionOfChildProjects = ko.observableArray();
+            self.visible = {
+                childProjects: ko.observable(true),
+            };
+
+            const resourceId = ko.unwrap(self.reportMetadata).resourceinstanceid;
+            const relatedResources = async() => {
+                const result = await reportUtils.getRelatedResources(resourceId);
+                const rrs = result?.related_resources;
```

Comment:
Avoid using abbrevations - that will cause confusion.  Longer variable names are OK as long as it's clear what you're refering to.  saying const relatedResources is preferred here.  Your function name should probably be changed to something like loadRelatedResources - since the function is performing some task, and it's unclear what task that is from the name alone.